testsuite/oxigraph-tests/geosparql/sf_equals.srx

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0"?>
+<sparql xmlns="http://www.w3.org/2005/sparql-results#">
+    <head>
+        <variable name="left"/>
+        <variable name="right"/>
+        <variable name="result"/>
+    </head>
+    <results>
+        <result>
+            <binding name="left">
+                <literal datatype="http://www.opengis.net/ont/geosparql#wktLiteral">Point(-83.38 33.95)</literal>
+            </binding>
+            <binding name="right">
+                <literal datatype="http://www.opengis.net/ont/geosparql#wktLiteral">Point(-83.38 33.95)</literal>
+            </binding>
+            <binding name="result">
+                <literal datatype="http://www.w3.org/2001/XMLSchema#boolean">true</literal>
+            </binding>
+        </result>
+        <result>
+            <binding name="left">
+                <literal datatype="http://www.opengis.net/ont/geosparql#wktLiteral">Point(-83.38 33.95)</literal>
+            </binding>
+            <binding name="right">
+                <literal datatype="http://www.opengis.net/ont/geosparql#wktLiteral">Point(0 1)</literal>
+            </binding>
+            <binding name="result">
+                <literal datatype="http://www.w3.org/2001/XMLSchema#boolean">false</literal>
+            </binding>
+        </result>
+        <result>
+            <binding name="left">
+                <literal datatype="http://www.opengis.net/ont/geosparql#wktLiteral">Point(-83.38 33.95)</literal>
+            </binding>
+            <binding name="right">
+                <literal datatype="http://www.opengis.net/ont/geosparql#geoJSONLiteral">{"type": "Point", "coordinates": [-83.38,33.95]}</literal>
```

Comment:
A GeoJSON will be a xsd:string literal? Or a RDF of it following the JSON-LD specs?

Reply:
[According to GeoSPARQL](https://docs.ogc.org/is/22-047r1/22-047r1.html#_rdfs_datatype_geogeojsonliteral) it is a RDF literal with datatype http://www.opengis.net/ont/geosparql#geoJSONLiteral like `"{\"type\": \"Point\", \"coordinates\": [-83.38,33.95]}"^^geo:geoJSONLiteral`